README.md

```diff
@@ -1,33 +1,32 @@
+During inference, the attribute (input/output/catalyst/regulation) is broken down into its individual <b>PhysicalEntity</b> (PE) instances. These PE instances are each run through the <a href="https://github.com/reactome/release-orthoinference/tree/develop/src/main/java/org/reactome/orthoinference/OrthologousEntityGenerator.java">createOrthoEntity</a> method. This method infers according to PE type: <i>GenomeEncodedEntity/EntityWithAccessionedSequence, Complex/Polymer, EntitySet or SimpleEntity</i>. In cases where the PE itself contains multiple <i>GKInstance</i> attributes (eg: Complexes <i>hasComponent</i> attribute, EntitySets <i>hasMember</i> attribute), these too are run through the createOrthoEntity method and inferred. Through this system, PE instances will be recursively broken down until they reach the <b>EntityWithAccessionedSequence</b> (EWAS) level and are inferred in the <a href="https://github.com/reactome/release-orthoinference/tree/develop/src/main/java/org/reactome/orthoinference/EWASInferrer.java">inferEWAS</a> method.
 
-After all valid ReactionlikeEvents instances have been inferred for a species, the final step is to populate the <b>Pathway</b> instances these RlEs are found in. Pathway inference takes place in its entirety in the <a href="https://github.com/reactome/data-release-pipeline/blob/develop/orthoinference/src/main/java/org/reactome/orthoinference/HumanEventsUpdater.java">HumanEventsUpdater</a> class, and involves creating the hierarchy structure from the Human Pathway equivalent (this will be further expanded at a future date).
+After all valid ReactionlikeEvents instances have been inferred for a species, the final step is to populate the <b>Pathway</b> instances these RlEs are found in. Pathway inference takes place in the <a href="https://github.com/reactome/release-orthoinference/tree/develop/src/main/java/org/reactome/orthoinference/PathwaysInferrer.java">PathwaysInferrer</a> class, and involves creating the hierarchy structure from the Human Pathway equivalent (this will be further expanded at a future date).
```

Comment:
The 'this will be further expanded at a future date' line can be removed, unless we still want to beef up the Pathway inference part of the README

Reply:
Done: https://github.com/reactome/release-orthoinference/pull/2/commits/6d7e54bfaa6b57879d76aeec66a8713efff68c19